applications/VeFunds.md

```diff
@@ -0,0 +1,99 @@
+Yes, a similar project is gitcoin but gitcoin does not provide full onChain management and didn't give authority to donators per project
+## Team 
+COGTECH
+### Team members
+
+- Emmanuel Akinroye
+
+### Team Website
+
+- https://cogtech.vercel.app
+
+### Team's experience
+
+I am a fullstack engineer with over half a decade years of experience, I have deployed various projects on blockchain. Many of which cann't be shared without the consent of the project owner. But I have developed the simplest version of this project and the git link to that is [https://github.com/Emma66886/blog.git](https://github.com/Emma66886/veOxFund) if you want to access this, kindly email me at emanuakinwumi@gmail.com so I can grant acccess to this private repo.
+the live website for the project is at https://vefunds.vercel.app
+An example blog I built on VeChain testnet https://lblog-ten.vercel.app/. 
+I have also deployed various projects which can be seen and accessed through my github link
+
+### Team Code Repos
+
+- https://github.com/Emma66886
+
+### Team LinkedIn Profiles
+
+- https://www.linkedin.com/in/emmanuel-akinroye-637747171/
+
+## Development Roadmap 
+- First Milestone(Platform Development)
+  Duration: three(3) months
+  UI/UX and more Research:1month
+  Front End And Backend and Mobile App Development :2months 
+  Cost: $9,000
+
+- Second Milestone(Testings and various adjustments)
+- Testing and community feedback : 1month
+  Duration:1month
+  Cost: $1,000
+
+-Third Milestone(Platform full lunch and maintenance)
+ Duration:1month
+ cost:$5,000
+
+ Total Cost($15,000)
+
+#### Overview
+
+|  | Milestone 1 | Milestone 2 | Total |
+| - | - |- | - |
+| Estimated Duration | 90 d | 30 d | 30 d |
+| Full-time equivalent (FTE) | 3 | 1 | 1 |
+| Cost ($ 15,000) | $ 9,000 | $ 1,000 | $ 5,000|
+
+#### Milestone 1 — Smart Contract & Backend
+
+| Number | Deliverable | Specification |
+|-|-|-|
+| 0a.| License | Apache 2.0 / MIT / Unlicense |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that can interact with the deployed smart contracts and backend service. |
+| 0c. | Testing Guide | The code will have proper unit-test coverage (e.g. 90%) to ensure functionality and robustness. In the guide, we will describe how to run these tests |
+| 1 | Smart Contracts |We will develop a smart contract that will enable each user to create campaign and for community to fund each campaign and be able to vote for rlease of funds based on each milestones . We will open source the smart contract and upload the ABI to [B32](https://github.com/vechain/b32). |
+| 2 | Backend | "We will create a backend service that will handle user authentication, manage capigns, and perform several other offchain and onchain functionalities" |
+
+#### Milestone 2  —  Frontend
+| Number | Deliverable | Specification |
+|-|-|-|
+| 1 | User Guide | We will provide a user guide or a demo video to show what features have been done.  |
+...
```

Comment:
Please complete the full milestone info, based on your application there are three milestones.